docker/package/Dockerfile

```diff
@@ -0,0 +1,23 @@
+# SPDX-FileCopyrightText: 2020 TQ Tezos <https://tqtezos.com/>
+#
+# SPDX-License-Identifier: MPL-2.0
+
+FROM ubuntu:18.04
+RUN apt-get update && apt-get install -y libev-dev libgmp-dev libhidapi-dev m4 perl pkg-config \
+  debhelper dh-make devscripts autotools-dev python3 python3-distutils wget
+RUN apt-get install -y software-properties-common && add-apt-repository ppa:avsm/ppa && apt-get update && apt-get install -y opam
+ENV USER dockerbuilder
+RUN useradd dockerbuilder && mkdir /tezos-packaging
```

Comment:
Why do we need `dockerbuilder` user?

Reply:
`dh_make` needs a dedicated user (not root) to create the package